src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -348,15 +353,21 @@ public boolean canSend(ItemStack itemStack) {
         private final RayTraceResult rayTraceResult;
         private final Block startBlock;
         private final boolean isWood;
+        private final int maxRange;
 
         public Finder(EntityPlayer p, ItemStack s, Coord4D loc, RayTraceResult traceResult) {
+            this(p, s, loc, traceResult, -1);
```

Comment:
Is this constructor used anywhere? No point having it if not

Reply:
thats just the old constructor thats why i didn't removed it, i will remove it than.

Reply:
ok removed it.